ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetBlockBodyValidator.java

```diff
@@ -162,7 +190,7 @@ private boolean validateEthHash(
     final BlockBody body = block.getBody();
 
     final Bytes32 ommerHash = BodyValidation.ommersHash(body.getOmmers());
-    if (!validateOmmersHash(header.getOmmersHash(), ommerHash)) {
+    if (!validateOmmersHash(header, header.getOmmersHash(), ommerHash)) {
```

Comment:
why pass the object and one of its attributes?  we can just pass header right?

Reply:
I consider this, kind of breaks the expect, actual naming convention in the helper methods...nbd either way IMO. Could potentially just pass in the log string instead of header